### ISSUE TITLE
Color background and Foreground

### ISSUE THREAD

**Issue by GhostValk** [arybczak/ncmpcpp#230]
the new syntax for the background and foreground color was switched to current_item_prefix and current_item_suffix entries in the config file.

I'm trying to define a color for the background AND foreground for the current selected item, but i've only managed to set it for the background. Currently my lines are:

current_item_prefix = $(41_blue)
current_item_suffix = $(end) 

Which only changes the background color; i've tried adding another tag between them, but it only affects the background colors.

How can i achieve this?

**Comment by GhostValk** [comment]
Somebody?

**Comment by arybczak** [comment]
Most likely your standard line contains another foreground color that overrides the one you set in `current_item_prefix`.

**Comment by GhostValk** [comment]
What do you mean?

**Comment by arybczak** [comment]
I mean your song_list_format (or columns format if you use columns view).

**Comment by GhostValk** [comment]
Yeah, i know, i'm defining colors for every column in column format tag, but aren't they supposed to work simultaneous?

Otherwise, how can i have a by-column color assigned and at the same time a different one for the current selected song?

**Comment by arybczak** [comment]
You can't at the moment.

**Comment by GhostValk** [comment]
Thanks a lot, that was my question =)

Could it be done by making the calling of the current_item_prefix/suffix AFTER the column format so it overrides it?

**Comment by GhostValk** [comment]
Any update on this?

**Comment by pascalsiemsen** [comment]
I'd be interested in this as well. Coming from MOCP, I try to achieve something similar to:

Item |  Foreground | Background
------------ | -------------  | -------------
file | grey | default 
selected_file | white | dark_grey

Meaning a selected item changes its background and foreground. Mainly because the reverse colour is not very readable. Or am I missing something here?

![2019-02-10-122109_1600x900_scrot](https://user-images.githubusercontent.com/36855562/52533067-9733a300-2d2e-11e9-8e94-000bbbfbcbd3.png)
